### PR TITLE
Add Equinox v1.0.1

### PR DESCRIPTION
**Pull request by sal0max**
After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/equinox.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  url "https://github.com/rlxone/Equinox/releases/download/v#{version}/Equinox.dmg",
+      verified: "https://github.com/rlxone/Equinox/"
+  name "equinox"
+  desc "Create dynamic wallpapers"
+  homepage "https://equinoxmac.com/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (neutric) [inline Casks/equinox.rb:14]: `strategy :github_latest` comes at the cost of a limited API access and is only necessary if there are also pre-releases. If all releases are production-ready, which it looks like to me, then you can omit the whole `livecheck stanza and rely on the built-in `livecheck` for GitHub repos.

File: Casks/equinox.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (neutric) [inline Casks/equinox.rb:15]: Please add `depends_on macos: ">= :mojave"`.

File: Casks/equinox.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  app "Equinox.app"
+
+  zap trash: [
+    "~/Library/Containers/com.rlxone.equinox",
+    "~/Library/Application Scripts/com.rlxone.equinox",
```
Comment (neutric) [inline Casks/equinox.rb:20]: Please sort the `zap` entries alphabetically.

**Review by neutric — COMMENTED** [review]
Thanks for your contribution to `homebrew-cask`, @sal0max! My review is in the code. Take it with a grain of salt as I am not a maintainer.

**Comment by sal0max** [comment]
Thanks a lot for your input, @neutric !
Changed accordingly. 🙂

**Comment by bevanjkay** [comment]
Thank you @sal0max 🎉

**Pull request merged**